cli/src/lib.rs

```diff
@@ -24,14 +31,36 @@ struct Args {
     /// SQL file to execute
     #[clap(short, long, value_parser)]
     execute: Option<PathBuf>,
+
+    /// PATH to dump whole database
+    #[clap(short, long, value_parser)]
+    dump: Option<PathBuf>,
 }
 
-pub fn run() {
+pub fn run() -> Result<()> {
     let args = Args::parse();
 
     if let Some(path) = args.path {
         let path = path.as_path().to_str().expect("wrong path");
 
+        if let Some(dump) = args.dump {
+            block_on(async {
+                let storage = SledStorage::new(path).expect("failed to load sled-storage");
+                let schemas = storage.fetch_all_schemas().await?;
+                println!("{schemas:?}");
```

Comment:
It's so weired fetch_all_schemas returns empty Vector even if there is Table and Index for sure..